src/Controller.js

```diff
@@ -88,4 +114,14 @@ export default class Controller {
         };
       });
   }
+
+  _addPrefetchOnHover(el, id) {
+    this._hoverlisteners.push(hoverintent(el,
```

Comment:
The indentation here is hard to read, I'd expect one argument per line. As is, hard to figure out what Params are part of what function.